time stamp
float64 ego_vx
float64 ego_vy
float64 ego_vth
PointCloudSensor[] pcs
sensor_msgs/PointCloud2 augpc
